source/_integrations/bmw_connected_drive.markdown

```diff
@@ -31,20 +31,87 @@ ha_platforms:
   - sensor
   - switch
 ha_integration_type: integration
-ha_quality_scale: platinum
 ---
 
-The **BMW Connected Drive** {% term integration %} lets you retrieve data of your BMW vehicle from the BMW Connected Drive portal. You need to have a working BMW Connected Drive account and a Connected Drive enabled vehicle for this to work.
-
-The **BMW Connected Drive** {% term integration %} also works with (recent) Mini vehicles. You need to have a working Mini Connected account, and a Mini Connected enabled vehicle for this to work.
+The **BMW Connected Drive** {% term integration %} lets you retrieve data of your BMW or MINI vehicle from the MyBMW portal (previously BMW Connected Drive).
 
 {% note %}
 The {% term entities %} available in Home Assistant heavily depend on your vehicle's capabilities (model year, headunit, etc.). The integration will make sure all available car attributes are added as entities.
 {% endnote %}
 
+## Prerequisites
+
+You need to have an active MyBMW account with a connected car. For MINI vehicles, you register with MINI Connected.
+
 For compatibility with your BMW vehicle check the [bimmer_connected page](https://github.com/bimmerconnected/bimmer_connected) on GitHub.
 
-This integration provides the following platforms:
+{% include integrations/config_flow.md %}
+
+{% configuration_basic %}
+Username:
+  description: |
+    Username of your MyBMW/MINI Connected account.
+
+    &nbsp;
+    
+    **China**: Your username/phone number must be prefixed with `86`, i.e. `8612345678`.
+  required: true
+  type: string
```

Comment:
```suggestion
```
removing these, as `required` and `type` are not displayed with the configuration_basic tag